### PR TITLE
Feature - show sponsors on main page

### PR DESCRIPTION
**Pull request by IGORnvk**
Displayed the golden sponsor on the main banner and all sponsors in the 'sponsors' section. For logos I used random pictures for now to see how it looks. Also I altered the tests a bit so they would pass. For testing, migrate, seed and go to the home page.

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -25,9 +25,7 @@
 |
 */
 
-Route::get('/', function () {
-    return view('welcome');
-})->name('welcome');
+Route::get('/', [TeamsController::class, 'getSponsors'])->name('welcome');
```
Comment (v-stamenova) [inline routes/web.php:28]: I'd rather we have a `HomeController` or something, but I don't like the idea of the welcome page being from the teams controller.

**Review by v-stamenova — CHANGES_REQUESTED** [review]
I think the sponsors should be something closer to the design for the companies page. @TimKardol should take a look as well

**Review by TimKardol — CHANGES_REQUESTED** [review]

File: resources/views/welcome.blade.php [hunk 2]
```diff
@@ -78,6 +83,20 @@ class="absolute -top-48 right-0 h-[25rem] transform translate-x-1/2 translate-y-
...
                 </div>
+
+                @if(!$allSponsors->isEmpty())
+                    <div class="flex flex-wrap justify-center gap-12 mt-16">
+                        @foreach($allSponsors as $sponsor)
+                            <div class="border border-2 @if ($sponsor->sponsorTier->name == 'golden' && $sponsor->is_approved) border-gold block
+                            @elseif ($sponsor->sponsorTier->name == 'silver' && $sponsor->is_approved) border-silver block
+                            @elseif ($sponsor->sponsorTier->name == 'bronze' && $sponsor->is_approved) border-bronze hidden xl:block @endif rounded-lg">
+                                <a href="{{ $sponsor->website }}">
+                                    <img alt="{{ $sponsor->name }}" src="{{ $sponsor->logo_path }}" class="w-full h-full px-6 py-6">
```
Comment (TimKardol) [inline resources/views/welcome.blade.php:94]: Do the logos complete fill up the block or do they have 'whitespace' around them? I think its better if the boxes are the same size as the image/logo. Looks more professional.

**Review by TimKardol — CHANGES_REQUESTED** [review]
Basically approval ready. Just take a look at the one comment I added

**Comment by TimKardol** [comment]
@IGORnvk Are you available tomorrow to sit down and pair program this?

**Comment by IGORnvk** [comment]
> @IGORnvk Are you available tomorrow to sit down and pair program this?

Yes, I am. Sometime in the morning preferably)

**Review by TimKardol — APPROVED** [review]
Looks good!

**Pull request merged**